<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>مدت زمان (دقیقه)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="device_sl">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="currentText">
      <string notr="true">PU1</string>
     </property>
     <item>
      <property name="text">
       <string>PU1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PU8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESMA</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string> دستگاه:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPlainTextEdit" name="failure_desc_sl">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>توضیحات</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>علت توقف</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="insert_failue_sl">
     <property name="text">
      <string>ثبت دستگاه</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTimeEdit" name="start_time_sl">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="displayFormat">
      <string>HH:mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>زمان شروع</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="stop_reason_sl">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <item>
      <property name="text">
       <string>تنظیم قالب و پلیسه گیری</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>تعویض قالب</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>خرابی جک</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>خرابی دستگیره</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>مکث در هر دور</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>تست مواد</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>متفرقه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="JalaliDateEdit" name="date_edit"/>
   </item>
   <item row="3" column="0">
    <widget class="QSpinBox" name="duration_sl">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JalaliDateEdit</class>
   <extends>QLineEdit</extends>
   <header>add_failure_sl_dialog</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
